content/v2/certificates.markdown

```diff
@@ -136,3 +136,196 @@ Responds with HTTP 200, renders the certificate private key.
 ~~~json
 <%= pretty_print_fixture("/getCertificatePrivateKey/success.http") %>
 ~~~
+
+## Let's Encrypt: Purchase a certificate {#letsencrypt-purchase}
+
+    POST /:account/domains/:domain/certificates/letsencrypt
+
+Purchase a [Let's Encrypt](https://dnsimple.com/letsencrypt) certificate with DNSimple.
+
+<note>
+Please note that the domain must be [delegated](https://support.dnsimple.com/articles/pointing-domain-to-dnsimple/) to DNSimple.
+</note>
+
+The certificate is **free of charge**, but some features of the certificate may require to be subscribed to an higher plan.
+
+### Certificate name
+
+The default certificate name is `www`, which is valid for the naked domain as well.
```

Comment:
I'd rather not call it `naked` domain. I always call it `APAX (root) domain`

Reply:
Do you mean _APEX_? 😉  They are actually all quite technical names. I generally use _root domain_. But an example is better.

> The default certificate name is `www` and covers both the root domain (e.g. `example.com`) and the `www` subdomain (e.g. `www.example.com`).